### PR TITLE
Update portfolio content and styling.

### PR DESCRIPTION
**Pull request by JPeterD**
Add basic content and styling to website and add a navigation bar.

TODO: 
Display random facts about me on portfolio website.

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -13,16 +13,21 @@
...
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+  // Pick a random fact.
+  const fact = facts[Math.floor(Math.random() * facts.length)];
 
   // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+  const factContainer = document.getElementById('fact-container');
```
Comment (tvenkata-sps) [inline portfolio/src/main/webapp/script.js:31]: I see you have removed the `greeting-container` and added the `fact-container` in the js. But in the html I don't see the  `fact-container` element.

Comment (JPeterD) [reply portfolio/src/main/webapp/script.js:31]: I planned to use the element later but not utilise it right now. So should I remove it for right now?

Comment (tvenkata-sps) [reply portfolio/src/main/webapp/script.js:31]: Ohh, Just add it to the PR description. It should be good.

Comment (JPeterD) [reply portfolio/src/main/webapp/script.js:31]: Alright, great added a todo comment.

**Review by tvenkata-sps — APPROVED** [review]
I don't see the Todo comment you mentioned earlier. I think after changing you need to push the pr again.

**Pull request merged**